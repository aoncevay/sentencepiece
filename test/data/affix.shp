a
a
a
aiki
áiki
ai
ai
ain
nko
ainko
ain
aitian
anan
ananan
anaan,naan
enaan,naan
inaan,naan
onaan,naan
naan
nan
an
anan
ananan
anan
ax
x
bain
bo
bait
baon
boan
boon
bata
bé
be
jo
be-
beiran
bekon
bi
bix
bicho
bira
bires
bo
bo-
cha-
cha
chaa
chi
chi-
tsi-
é
i
iba
iba
ibat
in-
ina
inat
isi
osi
jan-
jin-
jon-
ka-
kain
kan
kan
kas
kati
katit
káto
kawan
kaya
ke-
ke
kean
ken
ketian
ki-
ki
ki
kiin
kin
kin
kiran
ko-
koma
kon
ma-
ma
ma
me-
meax,mea
ainkoniax,ainkonia
keax,kea
ainoax,aiona
nkoniax,nkonia
oax,oa
nimeax
main
mein
men
mis
an
en
in
man
n
kan
ten
tan
nin
ton
tonin
n
n
na
na
na-
ne-
ni
no-
non
banon
non
nontian
nonx
nox
noxon
o
yo
sho 
oma
yoma
on
onmea
pa-
pacho
painon
pake
paket
pan
panon
pao
pari
patan
pe-
pi-
pon-
ra-
ra
a
rabe
ran-
re-
res
ri
riba
ribi
rin
ronki
ki
onki
ronkiki
shaman
shin
shoko
shoko
ska
skat
keet,kee,eet,ee
kiit,kii,iit,ii
koot,koo,oot,oo
kaat,kaa
meet,mee
t
ta-
ta
tan
tan-
taanan
tanan
tani
te-
ti
tian
tibi
tiibi
tio
toshi
toshit
tsi
tsi
wan
we
wen
wetsa
wi-
xan-
xe-
xo-
xon
xon
ya
a
sha
yama
áma
yantan
yá
yat
yon
yora
hira,ira
yosma
osma
isma