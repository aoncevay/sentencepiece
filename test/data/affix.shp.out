a ai ki ▁ki n i bo ra ▁bo ▁ra ti kan ribi kati kin ya ke ▁ke bi ma we ▁ma kas xon yama ax shoko nin rin bires ni ketian man nko res non riba ain baon tani ronki shaman o anan yora pari banon bicho wetsa an be ri ▁be cha kea pao wan ▁cha onki yoma bekon noxon ainoax aitian en oa ta ▁ta iba kon mea chaa ibat kain koma meet tiibi yantan in jo ▁in ▁te isi nox oax oma tan tio tsi ▁tan ▁tsi bata kaya kean meax rabe katit paket panon tonin toshi yosma nkonia painon taanan nkoniax t x é bé ee ii na on oo yo yá ▁ka ▁ko ▁me ▁na ▁ne ▁no ▁pa ▁pe ▁pi ▁re ▁wi ▁xe ▁xo bix chi eet iit ina ira kaa kee ken kii koo mee men mis nan oot osi pan sha sho ska ten ton wen yat yon áma ▁chi ▁jan ▁jin ▁jon ▁pon ▁ran ▁xan aiki bain bait bira boan boon hira inat isma kaat keax keet kiin kiit koot káto main mein naan nonx osma pake shin skat tian tibi áiki ainko aiona anaan enaan inaan kawan kiran onaan onmea pacho patan tanan ananan beiran nimeax toshit nontian ronkiki ainkonia ainkoniax